### PR TITLE
Adapt tests to the dbms master

### PR DESCRIPTION
**Pull request by kemm**

### CODE REVIEW

File: dbms/formatter.py [hunk 1]
```diff
@@ -689,7 +689,6 @@ def format_replication_database(body: Json) -> Json:
         "relations": [
             format_replication_relation(col) for col in body["collections"]
         ],
-        "views": [format_view(view) for view in body["views"]],
```
Comment (olegrok) [inline dbms/formatter.py:692]: Seems `format_view` function could be also removed

File: dbms/relation.py [hunk 2]
```diff
@@ -955,6 +955,8 @@ def get_many(self, documents: Sequence[Union[str, Json]]) -> Result[List[Json]]:
         :raise dbms.exceptions.DocumentGetError: If retrieval fails.
         """
         handles = [self._extract_id(d) if isinstance(d, dict) else d for d in documents]
+        if len(handles) == 0:
+            return []
```
Comment (olegrok) [inline dbms/relation.py]: Is it an optimization? Or is it a fix for some problem?

File: dbms/formatter.py [hunk 3]
```diff
@@ -215,7 +217,7 @@ def format_relation(body: Json) -> Json:
         result["schema"] = body["schema"]
 
     # New in 3.10
-    if "computedValues" in body:
+    if "computedValues" in body and body["computedValues"] is not None:
```
Comment (olegrok) [inline dbms/formatter.py]: I guess you can simplify it to `body.get("computedValues") is not None`

File: dbms/database.py [hunk 4]
```diff
@@ -1015,6 +1015,8 @@ def create_relation(
         :raise dbms.exceptions.RelationCreateError: If create fails.
         """
         key_options: Json = {"type": key_generator, "allowUserKeys": user_keys}
+        if key_generator != "autoincrement" and (key_increment is not None or key_offset is not None):
```
Comment (olegrok) [inline dbms/database.py:1013]: Is it backport from upstream or something else?

File: tests/conftest.py [hunk 5]
```diff
@@ -51,7 +51,7 @@ class GlobalData:
 
 def pytest_addoption(parser):
     parser.addoption("--host", action="store", default="127.0.0.1")
```
Comment (olegrok) [inline tests/conftest.py]: Let's call it `dbhost` for consistency

File: tests/test_async.py [hunk 6]
```diff
@@ -55,7 +55,10 @@ def test_async_wrapper_attributes(db, col, username):
     assert repr(job) == f"<AsyncJob {job.id}>"
 
 
-def test_async_execute_without_result(db, col, docs):
+def test_async_execute_without_result(db, col, docs, cluster):
+    if cluster:
+        pytest.skip("FIXME: for some reason coordinator does not report pending jobs")
```
Comment (olegrok) [inline tests/test_async.py:60]: Have you checked that it's unrelated to `views` removal?

File: dbms/database.py (hunk 4 above)
Comment (olegrok) [reply dbms/database.py:1013]: There are no such changes in upstream

File: dbms/relation.py (hunk 2 above)
Comment (olegrok) [reply dbms/relation.py]: Single mode instance with replication version 2 works fine. This change can hide some issues. I'll remove it.

File: tests/test_document.py [hunk 7]
```diff
@@ -1059,7 +1059,6 @@ def test_document_delete_many(col, bad_col, docs):
     for error, doc in zip(results, docs):
         assert isinstance(error, DocumentDeleteError)
         assert error.error_code in {1202}
-        assert "document not found" in error.error_message
```
Comment (olegrok) [inline tests/test_document.py:1062]: Current message is "NotFound:"

File: dbms/formatter.py (hunk 1 above)
Comment (olegrok) [reply dbms/formatter.py:692]: Removed a bunch of unused code

File: dbms/database.py (hunk 4 above)
Comment (olegrok) [reply dbms/database.py:1013]: Well, let's stay it

File: tests/conftest.py (hunk 5 above)
Comment (olegrok) [reply tests/conftest.py]: Done

File: tests/test_async.py (hunk 6 above)
Comment (olegrok) [reply tests/test_async.py:60]: Don't check. Probably we need to investigate it in future

**Review by knazarov — APPROVED** [review]

**Pull request merged**